docs/howto/object-storage/s3/object-lock.md

```diff
@@ -0,0 +1,361 @@
+# Object Lock
+
+Object Lock is a feature in S3 that enables you to lock your objects
+to prevent them from being deleted or overwritten. This feature provides
+a storage solution based on the **Write Once, Read Many (WORM)** model,
+which can be helpful when you want to maintain data integrity,
+comply with regulations, or retain data for a specific period.
+
+> Are you looking to set the expiration/retention time for your
+> objects? Then you should check our [Object expiry](expiry.md) guide.
+
+## Object Lock modes
+
+Object Lock offers two methods for managing object retention:
+**Retention periods** and **Legal hold**.
+
+### Retention periods
+
+Retention periods specify the length of time that an object
+remains locked and protected from being overwritten or deleted.
+When you set a retention period, the object is safeguarded for
+the specified time period. You can set the retention period in
+either days or years, with a **minimum of one day** and
+no maximum limit.
+
+In addition to retention periods, you can choose the retention
+mode that applies to your objects, which can be either
+**Governance** or **Compliance**.
+
+#### Governance mode
+
+> Balances data protection with flexibility,
+> allowing only authorized users to perform any
+> necessary actions while effectively managing objects.
+
+Governance mode is ideal for scenarios where you want to prevent
+most users from deleting objects during a specific retention period,
+but still allow authorized users with special permissions to modify
+the retention settings or delete objects as needed.
+
+To override or remove governance-mode retention settings, users must
+have the `s3:BypassGovernanceRetention` permission. With this permission,
```

Comment:
In our environment, how do users gain this permission? How can they find out whether their user account has it?

Reply:
I've removed the governance mode mention as we need to work on that part.